packages/@aws-cdk/aws-stepfunctions-tasks/test/sagemaker/create-transform-job.test.ts

```diff
@@ -102,6 +102,10 @@ test('create complex transform job', () => {
   const task = new SageMakerCreateTransformJob(stack, 'TransformTask', {
     transformJobName: 'MyTransformJob',
     modelName: 'MyModelName',
+    modelClientConfig: {
+      invocationsMaxRetries: 1,
+      invocationsTimeout: cdk.Duration.seconds(2400),
```

Comment:
perhaps assign it to minutes to ensure that it's being appropriately translated into the expected seconds